cursor_test.go

```diff
@@ -442,3 +442,66 @@ func (s *RethinkSuite) TestCursorNextResponse_object(c *test.C) {
 	c.Assert(ok, test.Equals, true)
 	c.Assert(b, jsonEquals, []byte(`{"foo":"bar"}`))
 }
+
+func (s *RethinkSuite) TestCursorPeek_idempotency(c *test.C) {
```

Comment:
These tests look great but could you also look into adding a godoc example. If you need some help creating them have a look at https://github.com/dancannon/gorethink/blob/master/example_test.go#L7 or just send me a message.